test/integration_test/health_monitor_test.go

```diff
@@ -19,7 +19,7 @@ import (
 const (
 	// node offline timeout just above 4.5 minutes
 	// which is the max time stork could take to delete a app pod.
-	nodeOfflineTimeout = 271 * time.Second
+	nodeOfflineTimeout = 295 * time.Second
```

Comment:
Did performance degrade?

Reply:
No, health monitor switch period is still 4.5 mins.
Torpedo StopDriver() labels node to stop Portworx driver, but does not actually waits for driver to shutdown only grace period of 10 Secs for some reason. that's why this increase timeout is needed

Reply:
@Rohit-PX  I think we should fix in torpedo will need it eventually https://github.com/portworx/torpedo/blob/master/drivers/volume/portworx/portworx.go#L1374